WordPress/openverse-frontend
# Ensure that the attribution box is always visible

## Fixes


Fixes #1656 by @sarayourfriend 

## Description


This issue is caused by the `license_url` field missing for some search results. This PR changes the frontend to automatically use the generated license URL if the API response does not contain it.

I've made a separate PR (WordPress/openverse-api#888) to ensure the API sends the `license_url` every time using the same logic as the frontend. Having this information in the response would be beneficial for other consumers of the API.


## Testing Instructions

0. Check out the PR and start the dev server.
1. Search for 'dog' and apply the 'CC0' license filter.
2. Find [this image](https://search-production.openverse.engineering/image/3d96348f-5b9f-4ce7-8854-8480fa3bf1a0) in the results and click on it.
3. You should see the attribution box (it would not be visible before this PR).

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
